pkg/vendir/cmd/sync.go

```diff
@@ -126,6 +129,31 @@ func (o *SyncOptions) Run() error {
 		if err != nil {
 			return fmt.Errorf("Syncing directory '%s': %s", dirConf.Path, err)
 		}
+		absRoot, err := filepath.Abs(dirConf.Path)
```

Comment:
- can move this into if conditional
- also would be nice to call it absRoot => allowedAbsRoot